.circleci/config.yml

```diff
@@ -0,0 +1,171 @@
+version: 2
+workflows:
+  version: 2
+  tests:
+    jobs: &workflow_jobs
+      - node4:
+          filters: &all_commits
+            tags:
+              only: /.*/
+      - node6:
+          filters: *all_commits
+      - node8:
+          filters: *all_commits
+      - node9:
+          filters: *all_commits
+      - node10:
+          filters: *all_commits
+      - lint:
+          requires:
+            - node4
+            - node6
+            - node8
+            - node9
```

Comment:
Also drop node9.  You only need to test 6, 8, and 10.

Reply:
Node9 EOLs this month (June 2018) so we could probably do this now.

Reply:
https://github.com/GoogleCloudPlatform/nodejs-repo-tools/issues/128

Here, @kinwa91  can manually edit but an issue is on the tool that generates this.

Reply:
@alexander-fenster are these part of your scripts then if they aren't handled by repo-tools? @theacodes and I will likely be doing work to move your script to be part of synthtool

Reply:
My script just copies it from anther repository:
https://gist.github.com/alexander-fenster/8e28b40658fa517dc3c0873edc31e233#file-package-setup-js-L50
We just had a discussion with @JustinBeckwith about what to do with my script. I guess the good location for most of the stuff there is repo-tools, and I can start moving stuff to repo-tools. What does synthtool do?

Reply:
Plan is to talk with @theacodes about this tomorrow in the yoshi party.